lib/howl/ui/window.moon

```diff
@@ -24,6 +24,8 @@ placements = {
   below: 'POS_BOTTOM'
 }
 
+GTK_SUPPORTS_HIDDEN_TITLEBAR = Gtk.get_major_version! >= 3 and Gtk.get_minor_version! >= 4
```

Comment:
I guess that whether [gtk_check_version](https://developer.gnome.org/gtk3/stable/gtk3-Feature-Test-Macros.html#gtk-check-version) would be more appropriate here, as this won't work with say Gtk 4.0. You want to implement that?